programmers/프린터.js

```diff
@@ -0,0 +1,19 @@
+function solution(pri, loc) {
+  const idx = [];
+  let print = 0;
+
+  for (let i = 0; i < pri.length; i += 1) idx.push(i);
```

Comment:
```suggestion
  for (let i = 0; i < pri.length; i++) idx.push(i);
```
오히려 여기서는 plusplus를 사용해도 상관 없습니다.
[https://eslint.org/docs/rules/no-plusplus#allowforloopafterthoughts](url)